audio/oscillator_node.rs

```diff
@@ -173,5 +178,14 @@ impl AudioNodeEngine for OscillatorNode {
         }
     }
 
-    make_message_handler!(AudioScheduledSourceNode: handle_source_node_message);
+    fn message_specific(&mut self, message: AudioNodeMessage, sample_rate: f32) {
```

Comment:
Prefix this argument with an underscore (`_sample_rate`) to remove the build [warning](https://travis-ci.com/servo/media/jobs/196081527#L1034), please.